website/releases/release-0.15.0.md

```diff
@@ -0,0 +1,339 @@
+etc., now depend on new
+storage and I/O classes. If you’re using these classes directly in your applications, you need to change your
+integration code and usage. For more details, check out [this section](#hudi-storage-and-io-abstractions-1).
+
+#### Module Changes
+
+As part of introducing new storage and I/O abstractions and making core reader logic Hadoop-agnostic, this release
+restructures the Hudi modules to clearly reflect the layering. Specifically,
+
+- [`hudi-io` module](https://github.com/apache/hudi/tree/release-0.15.0/hudi-io) is added to for I/O related
+  functionality, and the Hudi-native HFile reader implementation sits inside this new module;
+- [`hudi-common` module](https://github.com/apache/hudi/tree/release-0.15.0/hudi-common) contains the core
+  implementation of the [Apache Hudi Technical Specification](https://hudi.apache.org/tech-specs) and is now
+  Hadoop-independent;
+- [`hudi-hadoop-common` module](https://github.com/apache/hudi/tree/release-0.15.0/hudi-hadoop-common) contains
+  implementation based on Hadoop file system APIs to be used
+  with [`hudi-common` module](https://github.com/apache/hudi/tree/release-0.15.0/hudi-common) on engines
+  including Spark,
+  Flink, Hive, and Presto.
+
+If you use [`hudi-common` module](https://github.com/apache/hudi/tree/release-0.15.0/hudi-common) as the dependency
+before and Hadoop file system APIs and implementations, you should include
+all three
+modules, [`hudi-io`](https://github.com/apache/hudi/tree/release-0.15.0/hudi-io), [`hudi-common`](https://github.com/apache/hudi/tree/release-0.15.0/hudi-common),
+and [`hudi-hadoop-common`](https://github.com/apache/hudi/tree/release-0.15.0/hudi-hadoop-common), as the dependency
+now.
+
+#### Lock Provider API Change
+
+The [`LockProvider`](https://github.com/apache/hudi/blob/release-0.15.0/hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/transaction/lock/LockManager.java#L125)
+instantiation now expects
+the [`StorageConfiguration`](https://github.com/apache/hudi/blob/release-0.15.0/hudi-io/src/main/java/org/apache/hudi/storage/StorageConfiguration.java)
+instance as the second argument of the constructor. If you
+extend [`LockProvider`](https://github.com/apache/hudi/blob/release-0.15.0/hudi-common/src/main/java/org/apache/hudi/common/lock/LockProvider.java)
+to implement a custom lock provider
+before, you need to change the constructor to match the aforementioned constructor signature. Here's an example:
+
+```java
+public class XYZLockProvider implements LockProvider<String>, Serializable {
+  ...
+
+    public XYZLockProvider(final LockConfiguration lockConfiguration, final StorageConfiguration<?> conf) {
+      ...
+    }
+  
+  ...
+}
+```
+
+### Breaking Changes
+
+The 0.14.0 release added support for deducing spurious log blocks with a new log block reader for MOR
```

Comment:
we had not called this out in our 0.14.0 release. and not sure if this is user facing. So, would prefer to remove this.

Reply:
I removed it.  I assume the user should not hit any format compatibility issue here so it's OK not to mention it in the release notes.

Reply:
+1 If its not user facing, we can remove this.